### PR TITLE
#SQL_Task. Nazar Lukyanovich

### PR DESCRIPTION
**Pull request by nazlukyanovich**
https://travis-ci.org/github/nazlukyanovich/db-assignments/builds/719258192

### CODE REVIEW

File: task/sql-tasks.js [hunk 1]
```diff
@@ -177,7 +272,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT
+        COUNT(ProductID) AS "TotalOfCurrentProducts",
+        SUM(Discontinued) AS "TotalOfDiscontinuedProducts"
```
Comment (AlexMokin) [inline task/sql-tasks.js]: it's better to write this query like this
notice that from-clause is optional.



```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```

File: task/sql-tasks.js [hunk 2]
```diff
@@ -199,7 +308,24 @@ async function task_1_14(db) {
...
+    let result = await db.query(`
+    SELECT
+        SUM(MONTH(OrderDate) = 1) AS "January",
+        SUM(MONTH(OrderDate) = 2) AS "February",
+        SUM(MONTH(OrderDate) = 3) AS "March",
+        SUM(MONTH(OrderDate) = 4) AS "April",
+        SUM(MONTH(OrderDate) = 5) AS "May",
+        SUM(MONTH(OrderDate) = 6) AS "June",
+        SUM(MONTH(OrderDate) = 7) AS "July",
+        SUM(MONTH(OrderDate) = 8) AS "August",
```
Comment (AlexMokin) [inline task/sql-tasks.js]: use count when you need to count records, it shows intent better

**Review by AlexMokin — COMMENTED** [review]
Looks good. See comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by nazlukyanovich** [comment]
Thanks for attention
Fixed

**Comment by AlexMokin** [comment]
accepted

**Pull request closed without merging**